coderd/database/migrations/000072_template_display_name.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE templates ADD COLUMN display_name VARCHAR(64) NOT NULL DEFAULT '';
```

Comment:
I think using `text` here should suffice, or is there a reason to limit this to 64 on the database side? (Tip, the underlying type for `varchar` is `text` in PostgreSQL, so there's no space savings to be had.)

PS. Would you mind adding a comment to this new field?

```sql
COMMENT ON COLUMN templates.display_name IS 'Display name does things...';
```

(I'm on a self-appointed quest to have our database schema documented.)

Reply:
Comment added.

Regarding the `text` vs. `varchar`, I set there a similar limit to `template.name`:  [link](https://github.com/coder/coder/blob/20018f82b880f1df5860119b6bea0aa4f01fc36c/coderd/database/dump.sql#L348).

I assume that the `display_name` is a visual, human-friendly representation of a template name, and it will replace `template.name` in the UI, hence a similar char limit which won't scramble CSS/divs :)